types/init_service_type.go

```diff
@@ -31,12 +31,13 @@ type InitService struct {
 
 // ResourceMeta defines single database resource meta.
 type ResourceMeta struct {
-	TargetMiners  []proto.AccountAddress // designated miners
-	Node          uint16                 // reserved node count
-	Space         uint64                 // reserved storage space in bytes
-	Memory        uint64                 // reserved memory in bytes
-	LoadAvgPerCPU uint64                 // max loadAvg15 per CPU
-	EncryptionKey string                 `hspack:"-"` // encryption key for database instance
+	TargetMiners        []proto.AccountAddress // designated miners
+	Node                uint16                 // reserved node count
+	Space               uint64                 // reserved storage space in bytes
+	Memory              uint64                 // reserved memory in bytes
+	LoadAvgPerCPU       uint64                 // max loadAvg15 per CPU
+	EncryptionKey       string                 `hspack:"-"` // encryption key for database instance
+	EventualConsistency bool                   `hspack:"-"` // TODO, eventual consistency test, need later refactor
```

Comment:
ConsistencyLevel float64 may be better

Reply:
Ok, good idea.

Reply:
This is a TODO from @xq262144 , which is currently implemented by calling `chain.Query` directly if the flag is set, i.e., the weakest eventual consistency. Need some more tinking in `kayak` later.